solr/core/src/java/org/apache/solr/cli/AssertTool.java

```diff
@@ -531,7 +530,7 @@ private static boolean isSolrRunningOn(String url) throws Exception {
   }
 
   private static boolean runningSolrIsCloud(String url) throws Exception {
-    try (final SolrClient client = new Http2SolrClient.Builder(url).build()) {
+    try (final SolrClient client = SolrCLI.getSolrClient(url)) {
```

Comment:
This single change will make `solr assert --cloud` resilient for `/solr` suffix on the URL, so can be nice to include anyway.

Reply:
Makes sense.   Also, since we have a SolrCLI.getSolrCLient, let's use it!